modules/custommutator/bech32.cpp

```diff
@@ -370,6 +370,21 @@ std::string Encode(Encoding encoding, const std::string& hrp, const data& values
     return ret;
 }
 
+/** Encode a Bech32 or Bech32m string without the checksum. */
+std::string EncodeNoChecksum(const std::string& hrp, const data& values) {
+    // First ensure that the HRP is all lowercase. BIP-173 and BIP350 require an encoder
+    // to return a lowercase Bech32/Bech32m string, but if given an uppercase HRP, the
+    // result will always be invalid.
+    for (const char& c : hrp) assert(c < 'A' || c > 'Z');
+
+    std::string ret;
+    ret.reserve(hrp.size() + 1 + values.size() + CHECKSUM_SIZE);
```

Comment:
There's extra bytes here we don't need.
```suggestion
    ret.reserve(hrp.size() + 1 + values.size());
```